catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/ShareableMetacardPolicyExtension.java

```diff
@@ -103,7 +103,7 @@ private KeyValueCollectionPermission isPermitted(
     List<KeyValuePermission> permissions = getPermissions(allPerms);
     Map<String, Set<String>> grouped = groupPermissionsByKey(permissions);
     if (Collections.disjoint(grouped.keySet(), SHARED_PERMISSIONS_IMPLIED)) {
-      return match; // ignore all but shareable permissions
+      return match; // ignore all but sharing permissions
```

Comment:
✏️  While in this case, "sharing" is the proper grammar, everywhere else this change occurs makes it sound very awkward. `Shareable` was picked because, like the terminology used on an interface (Iterable, Serializable, etc) it describes an aspect of virtually any object.